projects/go-style-guide/src/app/features/ui-kit/components/button-docs/button-docs.component.html

```diff
@@ -107,7 +107,7 @@ <h2 class="go-heading-5">Primary Button</h2>
           <go-button (handleClick)="testClick()" buttonDisabled="true">Disabled</go-button>
         </div>
         <div class="go-column go-column--33">
-          <go-button (handleClick)="testClick()" buttonIcon="work">With Icon</go-button>
+          <go-button (handleClick)="testClick()" buttonIcon="work"></go-button>
```

Comment:
shouldn't a `go-button` always have text? If it's just an icon button shouldn't it be a `go-icon-button` instead?

Reply:
@StevenUlmer ... yeah. Jared and I talked about this. We're keeping it this way because the designs are fundamentally different between the two. Yet, UX is still showing this style of button with only an icon in it. So for now, 🤷‍♂